dev/doc/xml-protocol.md

```diff
@@ -174,7 +174,7 @@ Moves current tip to `${stateId}`, such that commands may be added to the new st
   </union>
 </value>
 ```
-* Failure: If `stateId` is in an error-state and cannot be jumped to, `errorFreeStateId` is the parent state of ``stateId` that shopuld be edited instead. 
+* Failure: If `stateId` is in an error-state and cannot be jumped to, `errorFreeStateId` is the parent state of `stateId` that should be edited instead.
```

Comment:
This isn't related at all and you should not have hesitated to open a different PR for this fix. One-line fixes are generally merged so much quicker.